File: 30-days-of-code/Day07/Arrays.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    Arrays.py                                          :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2021/03/07 10:50:54 by alaamimi          #+#    #+#              #
#    Updated: 2021/03/07 14:11:40 by alaamimi         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

#!/bin/bash 

import sys

if __name__ == '__main__':
    n = int(input())
    arr = list(map(int, input.rstrip().split()))

    reverse_str = []
    for i in range(n):
        reverse_str.append(arr[n - i - 1])
output_str = ""
for i in range(len(reverse_str)):
    output_str += str(reverse_str[i]) + " "
print(output_str)

File: 30-days-of-code/Day07/Arrays.java
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   Arrays.java                                        :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: alaamimi <alaamimi@student.42.fr>          +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2021/02/25 22:07:39 by alaamimi          #+#    #+#             */
/*   Updated: 2021/02/25 22:07:43 by alaamimi         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

import java.util.Scanner;

public class Solution
{
	public static void main(String[] args)
	{
		Scanner scanInput = new Scanner(System.in); //Scan the input

		int len = scanInput.nextInt();
		int[] str = new int[len];
		int i;

		i = 0;
		while (i < len)
		{
			str[i] = scanInput.nextInt();
			i++;
		}
		int index;

		index = len - 1;
		while (index >= 0)
		{
			System.out.print(str[index] + " ");
			index--;
		}
		scanInput.close();
	}
}
